src/Containers/Article/Edit/editArticle.jsx

```diff
@@ -0,0 +1,294 @@
+import React, { Component } from 'react';
+import { PropTypes } from 'prop-types';
+import { connect } from 'react-redux';
+import { ImageSideButton, Block, addNewBlock, createEditorState, Editor } from 'medium-draft';
+import { convertToRaw } from 'draft-js';
+import mediumDraftImporter from 'medium-draft/lib/importer';
+import axios from 'axios';
+import LoadingBar from 'react-top-loading-bar';
+import { ToastContainer } from 'react-toastify';
+import mediumDraftExporter from 'medium-draft/lib/exporter';
+import { faFeather } from '@fortawesome/free-solid-svg-icons';
+import { getArticle, updateArticle } from '../../../actions/article';
+import getUser from '../../../actions/userInfo';
+import Sidebar from '../../../Components/SideBar/SideBar';
+import Navbar from '../../../Components/NavBar/NavBar';
+import Button from '../../../Components/Common/Button/Button';
+import 'medium-draft/lib/index.css';
+import './editArticle.scss';
+import 'react-toastify/dist/ReactToastify.css';
+
+let images = [];
+let spinner = false;
```

Comment:
@dusmel What are these variables doing outside of the class?

Reply:
I am using a third party package that doesn't give a way to put them inside, and on top of that, they have to have global scope thus my reason to put them outside.